src/WebJobs.Script/Diagnostics/Extensions/LoggerExtension.cs

```diff
@@ -379,5 +392,17 @@ public static void JobHostFunctionTimeoutNotSet(this ILogger logger)
         {
             _jobHostFunctionTimeoutNotSet(logger, null);
         }
+
+        public static void AutorestGeneratedFunctionApplication(this ILogger logger, string sanitizedAutorestJson)
+        {
+            string newLine = NewLine;
```

Comment:
instead of explicitly passing `newLine` prepend `sanitizedAutorestJson` with `\n`